src/ui/pages/SendTransaction/TransactionWarnings/TransactionMayFailWarning.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import ValidationErrorIcon from 'jsx:src/ui/assets/validation-error.svg';
+import { Spacer } from 'src/ui/ui-kit/Spacer';
+import { UIText } from 'src/ui/ui-kit/UIText';
+
+export function TransactionMayFailWarning() {
+  return (
+    <>
```

Comment:
This layout needs to be abstracted. It's used in two separate components which are supposed to look the same

Reply:
Agree, I requested review too soon, this should be a draft PR actually... I'll extract the common display logic into `TransactionWarning` component